install_config/configuring_sdn.adoc

```diff
@@ -142,7 +143,39 @@ networkConfig:
 *redhat/openshift-ovs-multitenant* for the *ovs-multitenant* plug-in, or
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 <4> Service IP allocation for the cluster
-====
+
+Alternatively, you can create a pod network with multiple CIDR ranges by
+adding separate ranges into the `clusterNetworks` field with the range and the `hostSubnetLength`. 
+
+Multiple ranges can be used at once, and the range can be expanded. Pods can
+also be moved from one range to another. Pod allocations occur in the order
+listed, then when the range is full, move to the next on the list.
```

Comment:
Nodes are assigned to the first empty cidr range in the list. Pods are assigned to nodes by the master there is no guarantee into which cidr range a pod will be assigned.

Reply:
It should be "Node allocations" not "Pod allocations" and then otherwise is fine.